.github/workflows/build-images.yml

```diff
@@ -233,6 +233,14 @@ jobs:
         run: ./scripts/ci/images/ci_build_ci_image_on_ci.sh
       - name: "Push CI image ${{ matrix.python-version }}:${{ env.GITHUB_REGISTRY_PUSH_IMAGE_TAG }}"
         run: ./scripts/ci/images/ci_push_ci_images.sh
+      - name: >
+          Push empty CI images to finish waiting jobs:
+          ${{ matrix.python-version }}:${{ env.GITHUB_REGISTRY_PUSH_IMAGE_TAG }}"
+        if: failure() || cancelled()
+        run: ./scripts/ci/images/ci_push_empty_ci_images.sh
```

Comment:
Why do we have this in place of the cancel-on-ci-build step?

Reply:
(Script to find recently released deps looks useful, I'm just not sure why we made this change)